crates/holochain_types/src/websocket.rs

```diff
@@ -62,6 +63,33 @@ impl std::fmt::Display for AllowedOrigins {
     }
 }
 
+impl JsonSchema for AllowedOrigins {
```

Comment:
This schema was wrong because it's a custom serialization - fixed with a manual implementation